mysql-test/suite/rpl/t/rpl_rewrt_db-slave.opt

```diff
@@ -1 +1,6 @@
-"--replicate-rewrite-db=test->rewrite" "--replicate-rewrite-db=mysqltest1 -> test" "--replicate-rewrite-db=x -> y" "--replicate-rewrite-db=database_master_temp_01->database_slave_temp_01" "--replicate-rewrite-db=database_master_temp_02->database_slave_temp_02" "--replicate-rewrite-db=database_master_temp_03->database_slave_temp_03"
+"--replicate-rewrite-db=test->rewrite"
+"--replicate-rewrite-db=mysqltest1->test"
+"--replicate-rewrite-db=x->y"
+"--replicate-rewrite-db=database_master_temp_01->database_slave_temp_01"
+"--replicate-rewrite-db=database_master_temp_02->database_slave_temp_02"
+"--replicate-rewrite-db=database_master_temp_03->database_slave_temp_03"
```

Comment:
This separation into newlines should be added into the cosmetic commit

Reply:
Will update this before push, after final review.